biz.aQute.resolve/src/biz/aQute/resolve/obr/plugin/ObrExporter.java

```diff
@@ -0,0 +1,85 @@
+package biz.aQute.resolve.obr.plugin;
+
+import static aQute.bnd.osgi.Constants.RESOLVE_EXCLUDESYSTEM;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.util.AbstractMap.SimpleEntry;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+
+import org.osgi.resource.Resource;
+
+import aQute.bnd.annotation.plugin.BndPlugin;
+import aQute.bnd.build.Project;
+import aQute.bnd.osgi.FileResource;
+import aQute.bnd.osgi.repository.XMLResourceGenerator;
+import aQute.bnd.service.export.Exporter;
+import aQute.lib.exceptions.Exceptions;
+import biz.aQute.resolve.Bndrun;
+import biz.aQute.resolve.RunResolution;
+
+@BndPlugin(name = "OBR Exporter")
+public final class ObrExporter implements Exporter {
+
+    static final String TYPE = "exporter.obr";
```

Comment:
I think the name should start with "bnd." like the other Bnd exporter names.

Reply:
Makes sense to be consistent with the current naming convention. will do.

Reply:
DONE ✔️